templates/admin/form_library_settings.html

```diff
@@ -3,32 +3,56 @@
 	{% block breadcrumbs %}
 	<ul class="breadcrumb">
 		<li><a href="{% url 'admin_profile_settings' %}">Organization Settings</a></li>
-		<li class="active">Form Library</li>
+		<li class="active">Form Library Labels</li>
 	</ul>
 	{% endblock %}
 
 	<!-- Sub navigation -->
 	<div class="sub-navigation">
 		<div class="sub-navigation-header">
-			<h4 class="page-title">Form Library Settings</h4>
+			<h4 class="page-title">Form Library Labels</h4>
 		</div>
 	</div>
 
 	<div class="bg-white p-10 pt-0">
 		<div class="mb-15">
 			{% include 'admin/sections/side-nav.html' %}
 		</div>
-        <div class="coming-soon-container">
-            {% include '../comingsoon.html' %}
-        </div>
+		<div>
+			<form id="formLibraryLabelsForm">
+				{% csrf_token %}
+				<div class="form-group row">
```

Comment:
Suggest keeping the sequence the same as the navbar as Beneficiary, Distribution, Training instead of Beneficiary, Training, Distribution

Reply:
Done